src/Tools/Source/RunTests/Program.cs

```diff
@@ -42,7 +44,9 @@ internal static int Main(string[] args)
                 cts.Cancel();
             };
 
-            return Run(options, cts.Token).GetAwaiter().GetResult();
+            var result = Run(options, cts.Token).GetAwaiter().GetResult();
+            CheckTotalDumpFilesSize();
```

Comment:
There were complains from the .NET CI Build team regarding the total dump sizes in roslyn-internal (see RE: Roslyn-internal CI archive size). I think that a similar situation can happen in roslyn. What do you think?

---
In reply to: [122364659](https://github.com/dotnet/roslyn/pull/20129#discussion_r122364659) [](ancestors = 122364659)